src/styles/pages/_Search.module.scss

```diff
@@ -18,4 +18,11 @@
 
 .load-more {
   text-align: center;
+
+  button {
+    &:focus {
```

Comment:
We should consider fixing this on the `Button` component so we don't have to override its focus styles.

Using the same hover and focus style like we do in the component works fine for `<a>` buttons because the page update clears focus state. But as we see here, it gets messy when the button is actually a `<button>` that doesn't go away on click. Maybe we could use a subtle outline on focus instead?

Reply:
I've added a subtle `outline` on `:focus` similar to what [Tailwind CSS](https://v1.tailwindcss.com/components/buttons) uses. What do you think? @apmatthews @theodesp